secv_guis/bimask_app/main_window.py

```diff
@@ -766,7 +770,7 @@ def _switch_img(self, step=1):
             success = self._handle_img_selection(nxt_item.text())
             if success:
                 self.file_lists.img_list.file_list.setCurrentItem(nxt_item)
-
+       
```

Comment:
Linting: are you adding whitespaces at the end of the file here? ideally it should end with a single newline, or nothing